_posts/2020-04-09-status-update.md

```diff
@@ -0,0 +1,183 @@
+We sent them beta versions of both the [NetworkingDsc][ps-networkingdsc] and [UpdateServicesDsc][ps-updateservicesdsc] PowerShell modules, puppetized!
+
+That will let them test out replacing profiles like this:
+
+```puppet
+define profile::network::windows_interface (
+  Array[Stdlib::IP::Address]    $ipaddress,
+  String[1]                     $interfacealias,
+  Enum['IPv4','IPv6']           $addressfamily = 'IPv4',
+  Optional[Stdlib::IP::Address] $gw_address = undef,
+){
+
+  include profile::network::windows_install_dsc_modules
+
+  dsc {"${name}-ip-address":
+    resource_name => 'IPAddress',
+    module        => 'NetworkingDsc',
+    properties    => {
+      ipaddress      => $ipaddress,
+      interfacealias => $interfacealias,
+      addressfamily  => $addressfamily,
+    },
+    require       => Class['profile::network::windows_install_dsc_modules'],
+  }
+
+  if $gw_address {
+    dsc {"${name}-default-gw":
+      resource_name => 'DefaultGatewayAddress',
+      module        => 'NetworkingDsc',
+      properties    => {
+        address        => $gw_address,
+        interfacealias => $interfacealias,
+        addressfamily  => $addressfamily,
+      },
+      require       => Class['profile::network::windows_install_dsc_modules'],
+    }
+  }
+}
+```
+
+With one using the beta modules, like this!
+
+```puppet
+define profile::network::windows_interface (
+  Array[Stdlib::IP::Address]    $ipaddress,
+  String[1]                     $interfacealias,
+  Enum['IPv4','IPv6']           $addressfamily = 'IPv4',
+  Optional[Stdlib::IP::Address] $gw_address = undef,
+){
+
+  dsc_ipaddress {"${name}-ip-address":
+    dsc_ipaddress      => $ipaddress,
+    dsc_interfacealias => $interfacealias,
+    dsc_addressfamily  => $addressfamily,
+  }
+
+  if $gw_address {
+    dsc_defaultgatewayaddress {"${name}-default-gw":
+      dsc_address        => $gw_address,
+      dsc_interfacealias => $interfacealias,
+      dsc_addressfamily  => $addressfamily,
+    }
+  }
+}
+```
+
+If you're interested in joining our beta program, which we will be launching in earnest after Easter, please [email us][dsc-email] ([dsc@puppet.com][dsc-email]) and let us know:
+
+1. What are the top three PowerShell modules with DSC resources you'd like to see Puppetized?
+2. Are you interested in joining our beta test group for this ongoing project?
+
+If you just wanna keep an eye on things, you can follow the repository on GitHub and the [parent epic in JIRA][iac-41]!
+
+## PowerShell 101 Training
+
+In addition to helping oversee the DSC work, Michael just wrapped up running the team through his [PowerShell 101 Course][pwshop], which focuses on helping with the "blinking cursor" problem:
```

Comment:
Again, feel free to ignore, but similar to my earlier comment, do you want to change the perspective seeing you're the author? Not something I feel overly strongly about, so feel free to ignore 😄

Reply:
This one I can definitely replace. 🙂